scripts/update_my_docker_images.sh

```diff
@@ -13,39 +10,48 @@ script_directory="$(perl -e 'use File::Basename;
  print dirname(abs_path(@ARGV[0]));' -- "$0")"
 cd "$script_directory" || exit
 
-# Get access to all of refinebio
+# Get access to all of refinebio.
 cd ..
 
 print_description() {
-    echo 'This script will re-build all refine.bio docker images and push them to'
-    echo 'the specified Dockerhub repo.'
+    echo 'This script will re-build all refine.bio docker images and push '
+    echo 'them to the specified Dockerhub repository.'
 }
 
 print_options() {
     cat << EOF
 There are two required arguments for this script:
--d specifies the Dockerhub repo you would like to deploy to.
+-d specifies the Dockerhub repository you would like to deploy to.
 -v specifies the version you would like to build. This version will passed into
     the Docker image as the environment variable SYSTEM_VERSION.
     It also will be used as the tag for the Docker images built.
 
-There is also one optional argument:
+There is also optional arguments:
 -a also build the affymetrix image
     (we normally don't because it is so intense to build)
+-b sets a remote Docker builder for amd64 image builds.
```

Comment:
We should simplify this, default to `adm64` and have optarg `-x` to target `arm64`. Though I also think we probably shouldn't enable this until all images can be built on `arm64`. Would like to hear your thoughts on this.

Reply:
Actually I am unsure what this option does functionally.